iis-wf/iis-wf-export-actionmanager/src/test/resources/eu/dnetlib/iis/wf/export/actionmanager/entity/dataset/default/output/dataset.properties

```diff
@@ -0,0 +1,52 @@
+targetColumn=body
+targetColumnFamily=result
```

Comment:
Haven't we agreed that the extension of files with expected values should be 'expectations'?